frontend/src/app/side-menu/side-menu.component.html

```diff
@@ -0,0 +1,48 @@
+<!-- Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+
+<mat-sidenav-container class="sidenav-container">
+  <mat-sidenav #drawer class="sidenav" fixedInViewport
+      [attr.role]="(isHandset$ | async) ? 'dialog' : 'navigation'"
+      [mode]="(isHandset$ | async) ? 'over' : 'side'"
+      [opened]="(isHandset$ | async) === false">
+    <mat-toolbar>Menu</mat-toolbar>
+
+    <button (click)="uploadFiles()">
+      Upload Files
+    </button>
+    <input type="file" #fileUpload id="fileUpload" name="fileUpload" multiple="multiple"  style="display: none;"/>
+
+    <mat-nav-list>
+      <h2>Datasets</h2>
+      <!-- <a mat-list-item href="#">Link 1</a> -->
+    </mat-nav-list>
+  </mat-sidenav>
+  <mat-sidenav-content>
+    <mat-toolbar color="primary">
+      <button
+        type="button"
+        aria-label="Toggle sidenav"
+        mat-icon-button
+        (click)="drawer.toggle()" >]
+
+        <mat-icon aria-label="Side nav toggle icon">menu</mat-icon>
+      </button>
+      <span>Sensor Visualizer</span>
+    </mat-toolbar>
+    <!-- Add Content Here -->
```

Comment:
Is this also a placeholder?
You can add TODO in code to signify that.

Reply:
Removed the <add content here> comment